### ISSUE TITLE
Option to import local playlists

### ISSUE THREAD

**Issue by AbhiramTS** [harmonoid/harmonoid#383]
Option to import a local playlist file(m3u or m3u8). I was unable to import it using the current implementation.

**Comment by senntore** [comment]
Yeah I want this feature as well, importing them(m3u files) is must for me, considering I use them all of the time. I don't want to make all of my playlist again.

**Comment by Neonalig** [comment]
As a temporary workaround, I've found that Harmonoid appears to have partial CLI support.
You can run `harmonoid.exe "c:\path\to\my\playlist.m3u8"` and it will open the playlist and begin playing it. Certainly no import feature, and a bit of a pain to use, but it does seem to work.

**Comment by alexmercerind** [comment]
> As a temporary workaround, I've found that Harmonoid appears to have partial CLI support.
> You can run `harmonoid.exe "c:\path\to\my\playlist.m3u8"` and it will open the playlist and begin playing it. Certainly no import feature, and a bit of a pain to use, but it does seem to work.

We have that to support opening files from the file explorer & context menu.
